Maple2.Model/Game/Market/UgcMarketItem.cs

```diff
@@ -45,7 +49,7 @@ public class UgcMarketItem : MarketItem {
         writer.WriteLong(SellerCharacterId);
         writer.WriteUnicodeString();
         writer.WriteUnicodeString(SellerCharacterName);
-        writer.WriteUnicodeString(string.Join(",", Tags + ", " + Look.Name));
+        writer.WriteUnicodeString(string.Join(",", string.Join(",", Tags) + ", " + Look.Name));
```

Comment:
bit confused by the additional `string.Join` here, isn't the 2nd param just a single result already?

Reply:
The additional string.Join was due to needing to write each out item in the Tags array.